src/tests/multihost/alltests/conftest.py

```diff
@@ -1062,6 +1062,53 @@ def remove_journalsssd():
     request.addfinalizer(remove_journalsssd)
 
 
+@pytest.fixture(scope='class')
+def sssdproxyldap_test(session_multihost, request):
+    """ Backup and restore sssd.conf """
```

Comment:
This doc string needs to be updated to the steps of this fixture

Reply:
Done